code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -492,3 +492,240 @@
 		if(L.heal_damage(heal_amt/parts.len, heal_amt/parts.len, null, BODYPART_ORGANIC))
 			M.update_damage_overlays()
 	return 1
+	
+
+/datum/symptom/vampirism
+	name = "Hemetophagy"
+	desc = "The host absorbs blood from external sources, and seemlessly reintegrates it into their own bloodstream, regardless of its bloodtype or how it was ingested. However, the virus also slowly consumes the host's blood"
+	stealth = 1
+	resistance = -2
+	stage_speed = 1
+	transmittable = 2
+	level = 9
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	prefixes = list("Porphyric ", "Hemo")
+	bodies = list("Blood")
+	var/bloodpoints = 0
+	var/maxbloodpoints = 50
+	var/bloodtypearchive
+	var/bruteheal = FALSE
+	var/aggression = FALSE
+	var/vampire = FALSE
+	var/mob/living/carbon/human/bloodbag
+		threshold_descs = list(
+		"Transmission 4" = "Host appears to die when falling into a coma.",
+		"Transmission 6" = "The virus aggressively assimilates blood, resulting in contiguous blood pools being absorbed by the virus, as well as sucking blood out of open wounds of subjects in physical contact with the host.",
+		"Stage Speed 7" = "The virus grows more aggressive, assimilating blood and healing at a faster rate, but also draining the host's blood quicker",
+	)
+	
+	
+	threshold_descs = list("Transmission 4" "The virus recycles excess absorbed blood into restorative biomass, healing brute damage."
+					"Stage Speed 7" "The virus grows more aggressive, assimilating blood and healing at a faster rate, but also draining the host's blood quicker"
+					"Transmission 6" "The virus aggressively assimilates blood, resulting in contiguous blood pools being absorbed by the virus, as well as sucking blood out of open wounds of subjects in physical contact with the host."
+					)
+
+
+/datum/symptom/vampirism/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.transmission >= 4)
+		bruteheal = TRUE
+	if(A.transmission >= 6)
+		aggression = TRUE
+		maxbloodpoints += 50
+	if(A.stage_rate >= 7)
+		power += 1
+	if((A.stealth >= 2) && (A.transmission >= 6) && A.process_dead) //this is low transmission for 2 reasons: transmission is hard to raise, especially with stealth, and i dont want this to be obligated to be transmittable
+		vampire = TRUE
+		maxbloodpoints += 50
+		power += 1
+	if(ishuman(A.affected_mob) && A.affected_mob.get_blood_id() == /datum/reagent/blood)
+		var/mob/living/carbon/human/H = A.affected_mob
+		bloodtypearchive = H.dna.blood_type
+		H.dna.blood_type = "U"
+
+/datum/symptom/vampirism/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/M = A.affected_mob
+	switch(A.stage)
+		if(1 to 4)
+			if(prob(5))
+				to_chat(M, "<span class='warning'>[pick("You feel cold...", "You feel a bit thirsty", "It dawns upon you that every single human on this station has warm blood pulsing through their veins.")]</span>")
```

Comment:
"You feel a craving for iron?", "you could use a sip of...blood?", "the thought of biting a neck crosses your mind..."

Reply:
ty